superset/utils.py

```diff
@@ -240,6 +241,55 @@ def dttm_from_timtuple(d):
         d.tm_year, d.tm_mon, d.tm_mday, d.tm_hour, d.tm_min, d.tm_sec)
 
 
+def decode_dashboards(o):
```

Comment:
Do you think using [jsonpickle](https://jsonpickle.github.io/) with custom [handlers](https://jsonpickle.github.io/api.html#jsonpickle-handlers-custom-serialization-handlers) for encoding/decoding would be more pythonic?

Reply:
I agree it would be more pythonic but as it turns out, jsonpickle is also succeptible to the RCE vulnerability.